lib/registration.pm

```diff
@@ -652,6 +652,24 @@ sub fill_in_registration_data {
     }
 
     process_scc_register;
+
+    # Some licenses (like HA) can be shown on sle 15 after registration
+    if (check_screen('scc-downloading-license', timeout => 5) && is_sle('15+')) {
+        my @tags = qw(license-agreement yast_scc-automatic-changes yast_scc-installation-summary yast_scc-pkgtoinstall);
+        while (1) {
+            assert_screen(\@tags, timeout => 240);
+            send_key $cmd{ok}     if match_has_tag('yast_scc-automatic-changes');
+            send_key $cmd{accept} if match_has_tag('yast_scc-pkgtoinstall');
+            if (match_has_tag('yast_scc-installation-summary')) {
+                send_key $cmd{finish};
+                last;
+            }
+            if (match_has_tag('license-agreement')) {
+                y2_logs_helper::accept_license;
+                send_key $cmd{next};
+            }
+        }
+    }
```

Comment:
this must be handled within `process_scc_register`. What you are starting here looks like the problem I fixed in `wait_boot` and installation/first_boot

Reply:
I will move the code. I first put it in `process_scc_register` but was not sure after :)

Reply:
Done, 2 new runs just to quickly verify (but nothing should change, as it's the same code just "relocated").

- Verification run: http://1b210.qa.suse.de/tests/6187
- Regression test: http://1b210.qa.suse.de/tests/6188